source/assets/js/preview.js

```diff
@@ -0,0 +1,49 @@
+import $ from 'jquery';
+
+$(function() {
+	let initialVariant = window.location.hash.substr(1),
+		reservedIds = [/* 'changelog', 'documentation'*/],
+		selectVariant = () => {
+			let currentVariant = window.location.hash.substr(1),
+				$currentVariant = currentVariant.length ? $('#' + currentVariant) : [],
```

Comment:
Cosmetics: `$([])` would be more stable as a fallback since `$currentVariant` would always be a jQuery collection then. Just in case you expect every jQuery method on `$currentVariant` without checking its type first.